week-4/D-methods/exercise-5.js

```diff
@@ -17,9 +17,27 @@ var coffeeMachine = {
     },
     insertedAmount: 0,
     insertMoney: function(amount) {
-
+    this.insertedAmount = amount;
```

Comment:
Nearly right! This was the brief:
> insertMoney takes an amount in parameter to add money in the coffee machine.

Your method doesn't add to the amount, it replaces it.